src/odb/test/cpp/TestAccessPoint.cpp

```diff
@@ -45,6 +47,10 @@ BOOST_AUTO_TEST_CASE( test_default )
     BOOST_TEST(ap->hasAccess());
     BOOST_TEST(ap->getHighType() == dbAccessType::HalfGrid);
     BOOST_TEST(ap->hasAccess(dbDirection::DOWN));
+    auto path_segs = ap->getSegments();
+    BOOST_TEST(path_segs.size() == 2);
+    BOOST_TEST(path_segs[0].xMax() == 30);
+    BOOST_TEST(path_segs[1].yMax() == 80); 
```

Comment:
Rect overloads == operator. so you can just compare with a whole rect instead of one co-ordinate.

Reply:
Done.